Makefile

```diff
@@ -0,0 +1,18 @@
+OTELCOL_BUILDER_VERSION ?= 0.1.1
+OTELCOL_BUILDER_DIR ?= ~/bin
+OTELCOL_BUILDER ?= $(OTELCOL_BUILDER_DIR)/opentelemetry-collector-builder
+
+build: otelcol-builder
+	@$(OTELCOL_BUILDER) --config manifest.yaml
+
+otelcol-builder:
+ifeq (, $(shell which opentelemetry-collector-builder > /dev/null 2>&1))
```

Comment:
Would it be a good idea to suffix the executable with the version - otherwise when running locally, we could already have an older version and not try to download the requested version?

Reply:
It's on purpose: if there's a local version, we use that. This is quite useful for development purposes.

Reply:
Actually, a bug is causing the binary to always be downloaded. I'll fix that, it should try to use a local version if available.

Reply:
Is there any way it could output the version of the builder it is using - just to avoid any potential misunderstanding of which version is being used?

Reply:
The builder does print that when it first starts:

```
$ make build
2020-10-21T22:10:01.474+0200	INFO	cmd/root.go:81	OpenTelemetry Collector distribution builder	{"version": "0.1.1", "date": "2020-10-21T19:29:30Z"}
2020-10-21T22:10:01.474+0200	INFO	cmd/root.go:97	Using config file	{"path": "manifest.yaml"}
2020-10-21T22:10:01.475+0200	INFO	builder/main.go:80	Sources created	{"path": "./_build"}
2020-10-21T22:10:01.508+0200	INFO	builder/main.go:97	Compiling
2020-10-21T22:10:01.908+0200	INFO	builder/main.go:103	Compiled	{"binary": "./_build/observatorium-otelcol"}
```

Reply:
Ok good :+1: